docs/docs/tutorials/get-started/customize-sign-in-experience.md

```diff
@@ -3,3 +3,49 @@ sidebar_position: 3
 ---
 
 # Customize sign-in experience
+
+In the "Get Started" tab, click the "Create" button on the right, and the browser will redirect to the Sign-in Experience tab.
+
+## Sign-in Preview
+
+:::tip Do you know...
+You can preview the result in real-time! No matter what you update on the left, the right-side UI will immediately update to synchronize the change.
+:::
+
+The customer may use the app under different conditions, including platform, language, and appearance. 
+
+Feel free to change them to see the result of your combination.
+
+## Branding
+
+In this area, you can define several main elements of brand exposé: color, logo, and slogan.
+
+- **Brand color** will affect the color of components, e.g., buttons and links.
+- **Enable dark mode** controls if the UI will have a "dark feeling" when Logto detects related system preference.
+- Other options are pretty straightforward, so we skip them here.
+
+:::note
+You may notice the auto-generated *Brand color (Dark)* is slightly different compared to the original *Brand color*.
+
+Because Logto just took care of the visual difference between light and dark and computes a brand color exclusively for dark mode. If you are unsatisfied with the color, you are also free to customize it.
+:::
+
+## Terms of Use
+
+An app will likely have a Terms of Use in production, and it's mandatory to accept it before signing in or creating an account.
+
+Toggle on "Enable terms of use" and enter the URL to enforce this process. Logto will load it via an `<iframe>`.
```

Comment:
nit: better `Toggle on "Enable Terms of Use" and enter its content URL`?

Reply:
not sure since the wording in AC is not locked yet

Reply:
LOG-3069